tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1485,3 +1518,33 @@ def _test_update_with_queue_params(pcluster_config_reader, cluster, remote_comma
             "INFO: service\\[slurmctld\\] restarted",
         ],
     )
+
+
+def _test_memory_based_scheduling_enabled_false(
+    pcluster_config_reader,
+    cluster,
+    remote_command_executor,
+    slurm_commands,
+):
+    """Test Slurm without memory-based scheduling feature enabled"""
+
+    # check that memory-based scheduling is disabled by default
+    assert_that(slurm_commands.get_conf_param("SelectTypeParameters")).is_equal_to("CR_CPU")
+    assert_that(slurm_commands.get_conf_param("ConstrainRAMSpace")).is_equal_to("no")
+
+    # TODO: Add functional tests for memory-based scheduling
+
+
+def _test_memory_based_scheduling_enabled_true(
+    pcluster_config_reader,
+    cluster,
+    remote_command_executor,
+    slurm_commands,
+):
+    """Test Slurm with memory-based scheduling feature enabled"""
+
+    # check that memory-based scheduling is now enabled
+    assert_that(slurm_commands.get_conf_param("SelectTypeParameters")).is_equal_to("CR_CPU_MEMORY")
+    assert_that(slurm_commands.get_conf_param("ConstrainRAMSpace")).is_equal_to("yes")
+
+    # TODO: Add functional tests for memory-based scheduling
```

Comment:
Same as above

Reply:
that's fine: we decided to publish the PR as soon as possible even without functional tests to start the review of the actual logic as soon as possible.